grader/src/test/java/edu/pdx/cs410J/grader/scoring/ProjectSubmissionOutputFileParserTest.java

```diff
@@ -234,9 +235,21 @@ public void parseTest1() {
 
 
   private ProjectSubmission parse(OutputFile file) {
+    ProjectSubmission projectSubmission = null;
+    try {
     String text = file.getText();
     ProjectSubmissionOutputFileParser parser = new ProjectSubmissionOutputFileParser(new StringReader(text));
-    return parser.parse();
+    projectSubmission= parser.parse();
+   
+       parser.parse();
+    } catch (ParseException e) {
+      e.printStackTrace();
+    } catch (IOException e) {
```

Comment:
I'm not sure that this is what you want to do.  Yes, the stack trace will be printed, but `null` will be returned from the method.  That will probably cause the test to fail, but perhaps not in the way that anyone would expect.